src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/UnnamedSymbolCompletionProvider_Conversions.cs

```diff
@@ -20,15 +20,19 @@ namespace Microsoft.CodeAnalysis.CSharp.Completion.Providers
     internal partial class UnnamedSymbolCompletionProvider
     {
         // Place conversions before operators.
-        private readonly int ConversionSortingGroupIndex = 1;
+        private readonly int _conversionSortingGroupIndex = 1;
 
         /// <summary>
         /// Tag to let us know we need to rehydrate the conversion from the parameter and return type.
         /// </summary>
         private const string RehydrateName = "Rehydrate";
-        private static readonly ImmutableDictionary<string, string> ConversionProperties =
+        private static readonly ImmutableDictionary<string, string> s_conversionProperties =
             ImmutableDictionary<string, string>.Empty.Add(KindName, ConversionKindName);
 
+        // We set conversion items' match priority to lower than default so completion selects other symbols over it when user starts typing.
+        // e.g. method symbol `Should` should be selected over `(short)` when "sh" is typed.
```

Comment:
ok.  i think i've convinced myself this is ok.  it helps ensure that these new items don't collide with years of muscle memory.  and it's likely they'll appear for a lot of cases, but be rarely used.  so lowering pri seems reasonable.